packages/react-liveness/jest.setup.ts

```diff
@@ -7,3 +8,8 @@ import '@testing-library/jest-dom';
 if (typeof window.URL.createObjectURL === 'undefined') {
   window.URL.createObjectURL = jest.fn();
 }
+
+Object.defineProperty(window, 'TextDecoder', {
+  writable: true,
+  value: TextDecoder,
+});
```

Comment:
Do we need this for every test file? If not prefer declaration in the test file needed for unit test eff, _jest.setup.ts_ runs individually for each test file